omise/test/test_account.py

```diff
@@ -36,3 +46,57 @@ def test_retrieve(self, api_call):
         account.reload()
         self.assertEqual(account.id, 'acct_foo')
         self.assertRequest(api_call, 'https://api.omise.co/account')
+
+
+    @mock.patch('requests.patch')
+    def test_update(self, api_call):
+        account = self._makeOne()
+        class_ = self._getTargetClass()
+        self.mockResponse(api_call, """{
+          "object": "account",
+          "id": "account_test_no1t4tnemucod0e51mo",
+          "team": "acct_no1t4tnemucod0e51mo",
+          "livemode": false,
+          "location": "/account",
+          "country": "TH",
+          "currency": "THB",
+          "email": "somchai.prasert@example.com",
+          "created_at": "2019-12-31T12:59:59Z",
+          "supported_currencies": [
+            "THB",
+            "JPY",
+            "USD",
+            "EUR",
+            "GBP",
+            "SGD",
+            "AUD",
+            "CHF",
+            "CNY",
+            "DKK",
+            "HKD"
+          ],
+          "api_version": "2019-05-29",
+          "auto_activate_recipients": true,
+          "chain_enabled": true,
+          "zero_interest_installments": true,
+          "chain_return_uri": "https://dashboard.omise.co/test/settings",
+          "webhook_uri": "https://omise-flask-example.herokuapp.com/webhook",
+          "metadata_export_keys": {
+            "charge": [
+              "color",
+              "order_id"
+            ]
+          }
+        }""")
+
+        self.assertTrue(isinstance(account, class_))
+        account.chain_enabled = True
```

Comment:
I think we should re-write these a bit to make it clearer.

```python
self.assertFalse(account.chain_enabled)
account.update(chain_enabled=True)
self.assertEqual(account.chain_enabled, True)
```
Otherwise it looks like we just assign object's attribute directly (even though there's more stuff under the hood).